packages/components/aside/src/aside.js

```diff
@@ -65,7 +76,7 @@ customElementDefineHelper(
 					<div class="aside-note">
 						<slot name="note" class="note-in-aside"></slot>
 					</div>
-					<div class="aside-platform-object ${this.hasPlatformObject ? '' : 'hidden'}"">
```

Comment:
hmmm, our linter shouldn't stop this?

Reply:
Yes, it does stop. That is why I need to update it. I don't understand how does it been committed and merged.